daml/splice-dso-governance-test/daml/Splice/Scripts/TestOnboarding.daml

```diff
@@ -98,6 +102,56 @@ testOnboardValidator = do
       version = "0.2.0"
       contactPoint = "v1@example2.com"
     lastActiveAt = Some now
+    weight = None
+    kind = Some OperatorLicense
+  pure ()
+
+testGrantValidatorLicense : Script ()
```

Comment:
This file is getting really large.

wdyt about splitting it into `TestSvOnboarding` and `TestValidatorOnboarding`?

Reply:
actually, if you go for the unit tests, as recommended below, then that might not be required anymore.